packages/quantic/force-app/main/default/lwc/quanticResultChildren/__tests__/quanticResultChildren.test.js

```diff
@@ -142,7 +142,7 @@ describe('c-quantic-result-children', () => {
   });
 
   describe('when there are no child results available in the collection', () => {
-    it('should not display any child results and display the no more child results message', async () => {
```

Comment:
When there are no children in the collection, we no longer show the message, no children results will be displayed.